deploy/hoptimator-pod.yaml

```diff
@@ -8,3 +8,11 @@ spec:
     image: docker.io/library/hoptimator
     imagePullPolicy: Never
     command: ["sleep", "infinity"]
+    volumeMounts:
+    - name: config-volume
+      mountPath: /etc/config
+  volumes:
+  - name: config-volume
+    configMap:
+      name: hoptimator-configmap
+  
```

Comment:
Are both a `deployment` and `pod` spec necessary?

Reply:
The deployment is for the operator, while the pod is basically a jump box. They are the same image, and it would be reasonable to combine them into one deployment (i.e. we could just jump to one of the operators). Not sure which is more ergonomic yet.